pkg/apis/kops/v1alpha2/networking.go

```diff
@@ -152,6 +152,8 @@ type CalicoNetworkingSpec struct {
 	TyphaPrometheusMetricsPort int32 `json:"typhaPrometheusMetricsPort,omitempty"`
 	// TyphaReplicas is the number of replicas of Typha to deploy
 	TyphaReplicas int32 `json:"typhaReplicas,omitempty"`
+	// WireguardEnabled enables WireGuard encryption with Calico
```

Comment:
```suggestion
	// WireguardEnabled enables WireGuard encryption for all on-the-wire pod-to-pod traffic
	// (default: false)
```

Reply:
I resolved in https://github.com/kubernetes/kops/pull/10037/commits/51e7f2a691887571067d9e58156dfeff41544452 .